src/main/java/com/intellij/rt/debugger/agent/ClassTransformer.java

```diff
@@ -21,7 +21,15 @@ protected ClassLoader getClassLoader() {
     }
 
     public byte[] accept(ClassVisitor visitor, int parsingOptions, boolean storeClassForDebug) {
+        acceptOriginalByteCode(visitor, parsingOptions);
```

Comment:
A bit of grammar: it's bytecode, not byte code nor byte-code. So the proper naming should be `acceptOriginalBytecode`.

See also: `java.io.FilenameFilter`, `java.util.UUID#timestamp`, …